frontend/src/pages/video/video-page-container.tsx

```diff
@@ -1,14 +1,146 @@
-import { FC } from 'common/types/types';
+import { ChannelBaseResponse, FC, UserBaseResponseDto } from 'common/types/types';
 import { VideoChatContainer } from 'components/video-chat/video-chat-container';
-
 import styles from './video-page.module.scss';
+import { Button, Icon, Loader } from '../../components/common/common';
+import { AppRoute, IconName } from '../../common/enums/enums';
+import { useAppDispatch, useAppSelector, useEffect, useLocation, useNavigate } from '../../hooks/hooks';
+import defaultAvatar from '../../assets/img/default-user-avatar.jpg';
+import { videoActions, channelActions } from '../../store/actions';
+import { VideoBaseResponseDto } from '../../common/types/video/video';
+import clsx from 'clsx';
+import { getReactBtnColor } from './common/helper/get-react-btn-color';
 
 const VideoPageContainer: FC = () => {
+  const dispatch = useAppDispatch();
+  const navigate = useNavigate();
+  const location = useLocation();
+  const videoId = location.pathname.replace('/video/', '');
```

Comment:
there is a special hook for getting route params, `useParams`, if I remember correctly

Reply:
done